app/utils/kms_utils.py

```
@@ -0,0 +1,187 @@
+        
+        if access_key_id and secret_access_key:
+            _aws_kms_client = boto3.client(
+                'kms',
+                aws_access_key_id=access_key_id,
+                aws_secret_access_key=secret_access_key,
+                region_name=region
+            )
+            logger.info(f"AWS KMS initialized for region: {region}")
+        else:
+            logger.error("AWS KMS credentials missing")
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)